gunicorn/config.py

```diff
@@ -1031,7 +1030,7 @@ class AccessLog(Setting):
     desc = """\
         The Access log file to write to.
 
-        "-" means log to stderr.
+        ``-`` means log to `stderr`.
```

Comment:
Another alternative: Activate the sphinx.ext.intersphinx extension and use the data role as :data:\`sys.stderr\`.